package.json

```diff
@@ -21,7 +21,7 @@
     "dependencies": {
         "@actions/core": "^1.10.0",
         "@actions/exec": "^1.1.0",
-        "run-matlab-command-action": "github:matlab-actions/run-command#v1.1.2"
+        "run-matlab-command-action": "github:matlab-actions/run-command#v1.2.0"
```

Comment:
Do these submissions affect the version numbers that we have documented? 
https://github.com/matlab-actions#:~:text=When%20you%20define%20your%20workflow%2C%20specify%20this%20action%20as%20matlab%2Dactions/setup%2Dmatlab%40v1

Reply:
Great question! They shouldn't in this case, that's an internal dependency for the project so that we can use the exported `matlab` library. It's the same as if we were updating another js library used internally. I needed to update it because only v1.2+ has `matlab.runCommand` (https://github.com/matlab-actions/run-command/blob/main/src/matlab.ts) which can take an array of startup options.

Reply:
Sounds good. Thanks for letting me know. :)